docs/api/stories.md

```diff
@@ -0,0 +1,497 @@
+addParameters({ ... });
+```
+
+#### Documentation-only MDX
+
+Typically, when you use Storybook MDX, you define stories in the MDX documentation is automatically associated with those stories. But what if you want to write Markdown-style documentation and have it show up in your Storybook?
+
+If you don't define stories in your MDX, you can write MDX documentation and associate it with an existing story, or embed that MDX as its own documentation node in your Storybook's navigation.
+
+If you don't define a `Meta`, you can write Markdown and associate with an existing story. See ["CSF Stories with MDX Docs"](../writing-docs/mdx).
+
+To get a "documentation-only story", in your UI, define a `<Meta>` as you normally would, but don't define any stories. It will show up in your UI as a documentation node:
+
+<div style="background-color:#F8FAFC">
+TODO: VET THIS IMAGE.
+</div>
+
+<center>
+  <img src="https://raw.githubusercontent.com/storybookjs/storybook/master/addons/docs/docs/media/mdx-documentation-only.png" width="100%" />
+</center>
+
+## MDX file names
+
+Unless you use a custom webpack configuration, all of your `MDX` files should have the suffix `*.stories.mdx`. This tells Storybook to apply its special processing to the `<Meta>` and `<Story>` elements in the file.
+
+Be sure to update your Storybook config file to load `.stories.mdx` stories, as per the [`addon-docs` installation instructions](https://github.com/storybookjs/storybook/tree/master/addons/docs#installation).
+
+### ArgTypes
+
+> NOTE: This API is experimental and may change outside of the typical semver release cycle
+
+ArgTypes are a first-class feature in Storybook for specifying the behaviour of [Args](../writing-stories/args). By specifying the type of an arg you constrain the values that it can take and can also provide information about args that are not explicitly set (i.e. not required).
+
+You can also use argTypes to “annotate” args with information that is used by addons that make use of those args, for instance to instruct the controls addons to render a color choose for a string-valued arg.
+
+The most concrete realization of argTypes is the [Props Table](props table) doc block. Each row in the table corresponds to a single argType, as well as the current value of that arg.
+
+<div style="background-color:#F8FAFC">
+TODO: image mentioned in SB 6.0 needs to be further vetted
+</div>
+
+## Automatic argType inference
+
+If you are using the Storybook [docs](../writing-docs) addon (installed by default as part of [essentials](../essentials/introduction)), then Storybook will infer a set of argTypes for each story based on the `component` specified in the [default export](#default-export) of the CSF file. 
+
+To do so, Storybook uses various static analysis tools depending on your framework.
+
+<div style="background-color:#F8FAFC">
+TODO: further vet this item
+</div>
+
+> If you are using React, Storybook uses …, [read details], [open issues]. 
```

Comment:
@shilman can you add these details here? We probably need one line per framework that is supported.

Reply:
- React - react-docgen / react-docgen-typescript - https://github.com/reactjs/react-docgen / https://github.com/styleguidist/react-docgen-typescript
- Vue - vue-docgen-api - https://github.com/vue-styleguidist/vue-styleguidist/tree/dev/packages/vue-docgen-api
- Angular - compodoc - https://compodoc.app
- Web-components - custom-element.json - https://github.com/webcomponents/custom-elements-json
- Ember - YUI doc - https://github.com/ember-learn/ember-cli-addon-docs-yuidoc#documenting-components

Reply:
Links added, marking this as resolved.